src/west/main.py

```diff
@@ -508,11 +508,8 @@ def main(argv=None):
         # latest version, with the same arguments that we were given.
         log.dbg("sys.executable:\"{}\" sys.argv[0]:\"{}\" argv:\"{}\"".format(
```

Comment:
I guess this debug log needs an update too. Maybe just log `sys.argv[0]` and `argv`.

Reply:
updated